polkadot-parachains/src/service.rs

```diff
@@ -687,7 +643,7 @@ where
 		polkadot_config,
 		collator_options,
 		id,
-		rpc_extension_builder,
+		|_| Ok(RpcModule::new(())),
```

Comment:
This seems not correct to me either if we want to support the encointer-rpcs.

Reply:
Ok. I'll check